ui/app/clinical/constants.js

```diff
@@ -71,7 +75,8 @@ Bahmni.Clinical.Constants = (function () {
         globalPropertyToFetchActivePatients: 'emrapi.sqlSearch.activePatients',
         adtPrivilege: "app:adt",
         adtForwardUrl: "../adt/#/patient/{{patientUuid}}/visit/{{visitUuid}}/",
-        certificateHeader: "Print Header"
+        certificateHeader: "Print Header",
```

Comment:
Would it be possible to make `Print Header` a translation instead of keeping it hardcoded? This would enhance localization and maintain consistency across translations.